tests/libvmi/network.go

```diff
@@ -21,6 +21,7 @@ package libvmi
 
 import (
 	kvirtv1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/kubevirt/tests/libnet"
```

Comment:
nit: Please keep a separation from the production imports.

Reply:
done